odoo/addons/base/views/res_partner_views.xml

```diff
@@ -288,9 +288,6 @@
                                 </kanban>
                                 <form string="Contact / Address">
                                     <sheet>
-                                        <!-- parent_id and type fields needed in attrs in base_address_city module which overwrites
-                                        _fields_view_get() of partner. It would be better to put those fields there but the web client
-                                        dosen't support when a field is displayed several times in the same view.-->
```

Comment:
Normally having twice the same field in the view became supported.  (why add this comment?)

Reply:
removed, done